nifi-frontend/src/main/frontend/apps/nifi-registry/proxy.config.mjs

```diff
@@ -16,7 +16,7 @@
  */
 
 const target = {
-    target: 'http://localhost:18080',
+    target: 'https://localhost:18443',
```

Comment:
I don't think this change should be made in the source repo. Was this included accidentally?

Reply:
Thanks @kevdoran I have reverted this.